scripts/installation/bin/yb-ctl

```diff
@@ -1060,6 +1065,13 @@ class ClusterControl:
 
             subparsers[cmd_name] = self.setup_base_parser(cmd_name, help=help)
 
+        # Commands that take -f or --force flag
+        force_commands = ["destroy", "stop", "stop_node", "restart"]
```

Comment:
These aren't the only commands.  There's also `wipe_restart`, `remove_node`, `restart_node`.

Reply:
noted